code/game/objects/structures/bedsheet_bin.dm

```diff
@@ -141,105 +156,121 @@ LINEN BINS
 	icon_state = "sheethop"
+	item_state = "sheethop"
 	item_color = "hop"
 	dream_messages = list("authority", "a silvery ID", "obligation", "a computer", "an ID", "a corgi", "the head of personnel")
 
 /obj/item/bedsheet/ce
 	name = "chief engineer's bedsheet"
 	desc = "It is decorated with a wrench emblem. It's highly reflective and stain resistant, so you don't need to worry about ruining it with oil."
 	icon_state = "sheetce"
+	item_state = "sheetce"
 	item_color = "chief"
 	dream_messages = list("authority", "a silvery ID", "the engine", "power tools", "an APC", "a parrot", "the chief engineer")
 
 /obj/item/bedsheet/qm
 	name = "quartermaster's bedsheet"
 	desc = "It is decorated with a crate emblem in silver lining.  It's rather tough, and just the thing to lie on after a hard day of pushing paper."
 	icon_state = "sheetqm"
+	item_state = "sheetqm"
 	item_color = "qm"
 	dream_messages = list("a grey ID", "a shuttle", "a crate", "a sloth", "the quartermaster")
 
 /obj/item/bedsheet/magician
 	name = "magician's cape"
 	desc = "A magician never reveals his secrets."
 	icon_state = "sheetmagician"
+	item_state = "sheetmagician"
 	item_color = "magician"
 	dream_messages = list("trickery", "crime", "a gullible mark", "an angry wizard", "pixie dust")
 
 /obj/item/bedsheet/brown
 	icon_state = "sheetbrown"
+	item_state = "sheetbrown"
 	item_color = "cargo"
 	dream_messages = list("brown")
 
 /obj/item/bedsheet/black
 	icon_state = "sheetblack"
+	item_state = "sheetblack"
 	item_color = "black"
 	dream_messages = list("black")
 
 /obj/item/bedsheet/centcom
 	name = "\improper CentCom bedsheet"
 	desc = "Woven with advanced nanothread for warmth as well as being very decorated, essential for all officials."
 	icon_state = "sheetcentcom"
+	item_state = "sheetcentcom"
 	item_color = "centcom"
 	dream_messages = list("a unique ID", "authority", "artillery", "an ending")
 
 /obj/item/bedsheet/syndie
 	name = "syndicate bedsheet"
 	desc = "It has a syndicate emblem and it has an aura of evil."
 	icon_state = "sheetsyndie"
+	item_state = "sheetsyndie"
 	item_color = "syndie"
 	dream_messages = list("a green disc", "a red crystal", "a glowing blade", "a wire-covered ID")
 
 /obj/item/bedsheet/cult
 	name = "cultist's bedsheet"
 	desc = "You might dream of Nar'Sie if you sleep with this. It seems rather tattered and glows of an eldritch presence."
 	icon_state = "sheetcult"
+	item_state = "sheetcult"
 	item_color = "cult"
 	dream_messages = list("a tome", "a floating red crystal", "a glowing sword", "a bloody symbol", "a massive humanoid figure")
 
 /obj/item/bedsheet/wiz
 	name = "wizard's bedsheet"
 	desc = "A special fabric enchanted with magic so you can have an enchanted night. It even glows!"
 	icon_state = "sheetwiz"
+	item_state = "sheetwiz"
 	item_color = "wiz"
 	dream_messages = list("a book", "an explosion", "lightning", "a staff", "a skeleton", "a robe", "magic")
 
 /obj/item/bedsheet/nanotrasen
-	name = "\improper Nanotrasen bedsheet"
```

Comment:
Why did you remove this exactly?

Reply:
Well the PR it was porting stuff from had no "\improper" in it, so I just assumed it was a typo, but someone told me it actually has a function so I will put it back.

Reply:
It's a byond text macro that pretty much forces improper nouns, without it and with capitalization byond assumes it's a proper noun,

`a Nanotrasen bedsheet` vs `the Nanotrasen bedsheet`